src/main/java/io/jenkins/plugins/thememanager/Theme.java

```diff
@@ -79,6 +88,7 @@ public static Builder builder() {
   public static class Builder {
     private List<String> cssUrls = emptyList();
     private List<String> javascriptUrls = emptyList();
+    private boolean blueOceanCompatible = true;
```

Comment:
No strong opinion, but I would rather prefer opt-in at the current stage

```suggestion
    private boolean blueOceanCompatible = false;
```

Reply:
already changed, saw your comment after i'd pushed